### PR TITLE
Admin tenant meta data mnrva 631

### PR DESCRIPTION
**Pull request by pras7430**
## JIRA:
See ticket here - https://jira.rax.io/browse/MNRVA-631

 ## Description:
tenant create metadata service added

 ## Testing:
no test cases added for it, we may need to add the interceptor for the admin project.
 ## Screenshots:
(if applicable)

### CODE REVIEW

File: projects/admin/src/app/_service/tenant-meta-data.service.ts [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+@Injectable({
+  providedIn: 'root'
+})
+export class TenantMetaDataService {
+  constructor(private http: HttpClient, private env: EnvironmentConfig) {}
+
+  /**
+   * Request to the admin create tenant metadata API 
+   */
+  createTenantMetaData(metadata: TenantMetaData) {
```
Comment (FarajiA) [inline projects/admin/src/app/_service/tenant-meta-data.service.ts:22]: Looks like the test is missing for this function.

**Review by FarajiA — APPROVED** [review]

File: projects/admin/src/app/_service/tenant-meta-data.service.ts (hunk 1 above)
Comment (pras7430) [reply projects/admin/src/app/_service/tenant-meta-data.service.ts:22]: ok let me try one more time,
as i have shown you in the call, facing an issue with the admin service test cases, Unable to hit the Request interceptor through the admin.

Comment (FarajiA) [reply projects/admin/src/app/_service/tenant-meta-data.service.ts:22]: Oh right I forgot this was related.
We created a ticket for this https://jira.rax.io/browse/MNRVA-635

**Comment by FarajiA** [comment]
This PR is complete

**Pull request merged**